draft-iab-protocol-maintenance.md

```diff
@@ -95,13 +95,22 @@ robustness principle to avoid interoperability issues.
 There is good evidence to suggest that many important protocols are routinely
 maintained beyond their inception.  In particular, a sizeable proportion of IETF
 activity is dedicated to the stewardship of existing protocols.  This document
-serves primarily as a record of the hazards inherent in applying the robustness
-principle and to offer an alternative strategy for handling interoperability
+serves primarily as a record of the hazards in applying the robustness principle
+too broadly, and offers an alternative strategy for handling interoperability
 problems in deployments.
 
-Ideally, protocol implementations never have to apply the robustness principle.
-Or, where it is unavoidable, use of the robustness principle is viewed as a
-short term workaround that needs to be quickly reverted.
+Ideally, protocol implementations that can be actively maintained never have
```

Comment:
This is a pretty big change in stance.  Perhaps that was deliberate, but I prefer this framing.

```suggestion
Ideally, protocol implementations can be actively maintained and never have
```

Reply:
I’m fine with that tweak—mainly I think it’s important to mention the active maintenance here, and having it be an “and” still achieves that while maintaining the strength of the statement.